webapp/app/model/form_data.py

```diff
@@ -160,12 +160,13 @@ class InputDataInvalidError(ValueError):
 
 class MandatoryFieldMissingValidationError(InputDataInvalidError):
     """Raised in case of a mandatory field missing"""
-    message = _l('form.lotse.input_invalid.mandatory_field_missing')
-
     def __init__(self, missing_fields=None):
         super().__init__()
         self.missing_fields = missing_fields
 
+    def get_message(self):
+        return _l('form.lotse.input_invalid.mandatory_field_missing', num_errors=len(self.missing_fields))
+
```

Comment:
You could just test this function by setting the missing fields and then checking that the message contains the number of errors. However, I do not think it is that important to do this.

Reply:
Won't do it then^^